.circleci/config.yml

```diff
@@ -62,13 +75,39 @@ jobs:
           command: |
             mkdir -p workspace/reports
       - run:
-          name: Generate all language libraries
+          name: Generate Pubsub Libraries
+          command: |
+            cd workspace/gapic-generator
+            python generate_clients.py \
+                google/pubsub/artman_pubsub.yaml \
+                --user-config=.circleci/artman_config.yaml \
+                --log=/tmp/workspace/reports/smoketest.log
+      - run:
+          name: Generate Logging Libraries
```

Comment:
Since this `run` task happens sequentially, if the previous `Generate Pubsub Libraries` task fails, then the `Generate Logging libraries` task will also fail. So if the python generation fails for pubsub, we don't get to see if logging fails for java.

What do you think about setting `when: always` to each of these run configs?